### PR TITLE
Remove user groups

### PR DESCRIPTION
**Pull request by raabdesselam**
Remove  code of user groups

### CODE REVIEW

**Comment by dduponchel** [comment]
You should remove the folder `app/src/core/usergroups` if it isn't used.

**Comment by dduponchel** [comment]
We still have references to user groups (`git grep -iE "user.?group"`), could you remove them ?

File: app/styles/main.scss [hunk 1]
```diff
@@ -1224,7 +1224,7 @@ table.fragment-list-creatives {
   }
 }
 
-table.fragment-list-user-groups {
+table.fragment-list-audience-groups {
```
Comment (dduponchel) [inline app/styles/main.scss]: You changed the class to `fragment-list-audience-segment`, not `fragment-list-audience-groups`.

**Comment by thomastosoni** [comment]
:+1:

**Pull request merged**